pkg/features/features.go

```diff
@@ -47,6 +48,13 @@ const (
 	// alpha: v0.1.6
 	PodPreset utilfeature.Feature = "PodPreset"
 
+	// CatalogRestrictions controls whether the ClusterServiceBroker will use
```

Comment:
Can we change the verbiage here around "control which services and plans have k8s resources created for them" ?

Reply:
that was your text! :D

Reply:
updated comment